hoomd/mpcd/stream.py

```diff
@@ -693,6 +693,115 @@ def set_params(self, R=None, boundary=None):
             self.boundary = boundary
 
         bc = self._process_boundary(self.boundary)
-        self._cpp.geometry = _mpcd.SphereGeometry(self.R,bc)
+        self._cpp.geometry = _mpcd.SphereGeometry(self.R,0.0,bc)
         if self._filler is not None:
             self._filler.setGeometry(self._cpp.geometry)
+
+class dryingsphere(_streaming_method):
```

Comment:
Why not name it like the C++ class?
```suggestion
class drying_droplet(_streaming_method):
```